packages/packages-groonga-org-package-task.rb

```diff
@@ -82,10 +84,15 @@ def download_packages(target_namespace)
       archive = File.expand_path(url.split("/").last)
       rm_f(archive)
       sh("wget", url)
-      cd(repositories_dir) do
-        sh("tar",
-           "xf", archive,
-           "--strip-components=#{built_package_n_split_components}")
+      if target_namespace == :windows
+        mkdir_p("#{repositories_dir}/windows/groonga/")
+        cp(archive, "#{repositories_dir}/windows/groonga/", {:preserve => true})
+      else
+        cd(repositories_dir) do
+          sh("tar",
+             "xf", archive,
+             "--strip-components=#{built_package_n_split_components}")
+        end
       end
       rm_f(archive)
```

Comment:
How about using existing variables instead of literals such as `windows` and `groonga`?

```suggestion
      case target_namespace
      when :apt, :yum
        cd(repositories_dir) do
          sh("tar",
             "xf", archive,
             "--strip-components=#{built_package_n_split_components}")
        end
        rm_f(archive)
      else
        download_dir = "#{repositories_dir}/#{target_namespace}/#{@package}"
        mkdir_p(download_dir)
        mv(archive, download_dir)
      end
```